tsconfig.json

```diff
@@ -13,11 +13,11 @@
     "downlevelIteration": true,
     "lib": [ "es2019", "dom", "ES2021.WeakRef" ],
     "types": [ "node" ],
-    "typeRoots": [ "./node_modules/@types" ]
+    "typeRoots": [ "./node_modules/@types" ],
+    "skipDefaultLibCheck": true,
```

Comment:
`skipDefaultLibCheck` will skip checking the types in `typescript/lib.*.d.ts`, and other type definitions in the `node_modules` are still checked

Reply:
https://www.typescriptlang.org/tsconfig/#skipDefaultLibCheck says

> Skip Default Lib Check - 
skipDefaultLibCheck

> Use skipLibCheck instead. Skip type checking of default library declaration files.

Where skipLibCheck says

> This can save time during compilation at the expense of type-system accuracy.

Reply:
It does seem to work though! The wonders of typescript. Super thanks for the PR, this will also be very helpful for the BBT transition.